packages/mantine/src/theme/Theme.tsx

```diff
@@ -412,10 +412,11 @@ export const plasmaTheme: MantineThemeOverride = createTheme({
         }),
         Tooltip: Tooltip.extend({
             defaultProps: {
-                maw: 300,
+                maw: 280,
                 multiline: true,
                 withArrow: true,
                 zIndex: 10000,
+                color: PlasmaColors.grape[9],
```

Comment:
```suggestion
                color: PlasmaColors.violet[9],
```
<img width="334" alt="image" src="https://github.com/user-attachments/assets/574294c5-c613-4217-a302-85a0d4260419" />